resolver/ng/osfamilymap.yaml

```diff
@@ -5,3 +5,7 @@ Debian:
       enabled: True
       remove: False
       file: /etc/resolvconf/run/resolv.conf
+
+FreeBSD:
+  ng:
+    group_name: wheel
```

Comment:
let's use the unified style where it's like this:
```
resolver:
  ng:
    user: root
    group: wheel
```

so in case we needed to change the user later no need to edit the state again

and the same for main `resolver` formula.

Reply:
Why would `user` and `group` go under `ng` rather than the root (`resolver`)?

Reply:
Yeah, here is actually the idea. The ng formula is not really splitted from the main formula.
Here I'm speaking about the idea itself not about `ng`.

This comment was actually a supplement of this:
> this should be resolver.ng or no need to add user/group under ng section.

What we need right now is simply as you did. So all good now.

Sorry for confusing.